streams/next.json

```diff
@@ -390,95 +390,96 @@
+                            "image": "ami-03eb4c9349a9d9d69"
                         },
                         "eu-west-1": {
-                            "release": "35.20211029.1.0",
-                            "image": "ami-08b78fbe423580888"
+                            "release": "35.20211107.1.0",
+                            "image": "ami-093a3c5098f40b20c"
                         },
                         "eu-west-2": {
-                            "release": "35.20211029.1.0",
-                            "image": "ami-06a3af4dedb07cb24"
+                            "release": "35.20211107.1.0",
+                            "image": "ami-07326d7cb019ac862"
                         },
                         "eu-west-3": {
-                            "release": "35.20211029.1.0",
-                            "image": "ami-003d9d47c1a774b47"
+                            "release": "35.20211107.1.0",
+                            "image": "ami-01513969d7e31d969"
                         },
                         "me-south-1": {
-                            "release": "35.20211029.1.0",
-                            "image": "ami-04dce8485874a7f2c"
+                            "release": "35.20211107.1.0",
+                            "image": "ami-055661f25762d8550"
                         },
                         "sa-east-1": {
-                            "release": "35.20211029.1.0",
-                            "image": "ami-0964137e0b22d7ccf"
+                            "release": "35.20211107.1.0",
+                            "image": "ami-0851bc4e0f4b559a3"
                         },
                         "us-east-1": {
-                            "release": "35.20211029.1.0",
-                            "image": "ami-0e640f10636a07ccd"
+                            "release": "35.20211107.1.0",
+                            "image": "ami-02b3bf24c78e8ac0e"
                         },
                         "us-east-2": {
-                            "release": "35.20211029.1.0",
-                            "image": "ami-021fded241cd019a1"
+                            "release": "35.20211107.1.0",
+                            "image": "ami-027bc181bc7f2f740"
                         },
                         "us-west-1": {
-                            "release": "35.20211029.1.0",
-                            "image": "ami-0ca3cdc279a461e72"
+                            "release": "35.20211107.1.0",
+                            "image": "ami-02a7373d46bf58cf5"
                         },
                         "us-west-2": {
-                            "release": "35.20211029.1.0",
-                            "image": "ami-0736351ad242a9868"
+                            "release": "35.20211107.1.0",
+                            "image": "ami-05a13bd1e186b87f9"
                         }
                     }
                 },
                 "gcp": {
+                    "release": "35.20211107.1.0",
```

Comment:
we should take note that `release:` is added here, just in case we see any failures in later steps (i.e. website)